### PR TITLE
Wallet id setup

### PR DESCRIPTION
**Pull request by ShahzadRiffat**
@maycon-mello please check this PR, Units tests are failing but i think these are not because of my changes. Please have a look.

### CODE REVIEW

File: src/features/accounts/AccountsScreen.js [hunk 1]
```diff
@@ -213,10 +218,16 @@ export const AccountsScreen = withErrorBoundary(
         </ScrollView>
         {isEmpty ? (
           <Footer marginBottom={114} marginLeft={26} marginRight={26} flex>
-            <BigButton onPress={onAddAccount} icon={<PlusCircleIcon />}>
+            <BigButton
+              testID="CreateNewAccount"
```
Comment (mike-parkhill) [inline src/features/accounts/AccountsScreen.js:222]: it's a minor thing but you're mixing the naming style. Some up above use underscores (receive_tokens_btn), others us camelCase (tokenReceive) and here it's Pascal case (CreateNewAccount). Can we pick one style and stick with it throughout?

File: src/features/wallet/CreateWalletScreen.js [hunk 2]
```diff
@@ -42,6 +43,7 @@ export function CreateWalletScreen() {
           variant="ghost"
           mt={4}
           testID="import_existing"
+          accessibilityLabel="import_existing"
```
Comment (mike-parkhill) [inline src/features/wallet/CreateWalletScreen.js:46]: previously you were setting `accessibilityID` but now it's `accessibilityLabel`. Why the change? What's the difference?

File: ios/Podfile.lock [hunk 3]
```diff
@@ -709,4 +709,4 @@ SPEC CHECKSUMS:
 
 PODFILE CHECKSUM: 35a4b05e323abe78110fcc3f3e07496820e99698
 
-COCOAPODS: 1.10.1
+COCOAPODS: 1.11.2
```
Comment (mike-parkhill) [inline ios/Podfile.lock:712]: is this upgrade intentional?

File: ios/Podfile.lock [hunk 4]
```diff
@@ -641,7 +641,7 @@ SPEC CHECKSUMS:
   CocoaAsyncSocket: 065fd1e645c7abab64f7a6a2007a48038fdc6a99
   DoubleConversion: cf9b38bf0b2d048436d9a82ad2abe1404f11e7de
   FBLazyVector: e686045572151edef46010a6f819ade377dfeb4b
-  FBReactNativeSpec: 5b40d13f01381fb5b95034ff589c98cbd8299c54
+  FBReactNativeSpec: 90dc7e6dd04f23a173ef8d5a29535c695400ae7c
```
Comment (mike-parkhill) [inline ios/Podfile.lock:644]: Does changing this ID affect anything?

**Review by mike-parkhill — CHANGES_REQUESTED** [review]

File: ios/Podfile.lock (hunk 3 above)
Comment (Riffat-Shahzad) [reply ios/Podfile.lock:712]: I have not done this, Maycon needs to check, how this happened.

File: src/features/wallet/CreateWalletScreen.js (hunk 2 above)
Comment (Riffat-Shahzad) [reply src/features/wallet/CreateWalletScreen.js:46]: Combination of accessibilityLabel makes it visible

File: ios/Podfile.lock (hunk 4 above)
Comment (Riffat-Shahzad) [reply ios/Podfile.lock:644]: some update has done this. I have done it explicitly. Reverting it

File: ios/Podfile.lock (hunk 3 above)
Comment (maycon-mello) [reply ios/Podfile.lock]: This is auto generated lock file, should be fine

**Comment by maycon-mello** [comment]
@mike-parkhill @ShahzadRiffat I did a small refactor adding a utility function to inject testIDs and accessibilityLabel properties. Also fixed some linting issues

**Comment by mike-parkhill** [comment]
> @mike-parkhill @ShahzadRiffat I did a small refactor adding a utility function to inject testIDs and accessibilityLabel properties. Also fixed some linting issues

That helper looks useful. Can we get it pulled through all the other changes? I'm still seeing a mix of `accessibilityLabel` and `accessibilityID` being used as well as the inconsistent naming that I mentioned earlier.

**Comment by maycon-mello** [comment]
> > @mike-parkhill @ShahzadRiffat I did a small refactor adding a utility function to inject testIDs and accessibilityLabel properties. Also fixed some linting issues
> 
> That helper looks useful. Can we get it pulled through all the other changes? I'm still seeing a mix of `accessibilityLabel` and `accessibilityID` being used as well as the inconsistent naming that I mentioned earlier.

accessibilityID looks wrong, I can fix that. Will also check to make sure we are using the helper in other places

**Comment by maycon-mello** [comment]
@mike-parkhill I've updated all the test IDs, it should be using the same naming convention now.

**Review by mike-parkhill — APPROVED** [review]

**Pull request merged**